README.md

```diff
@@ -235,15 +236,14 @@ We are also interested in assistance from code reviewers.
 
 Please contact Douglas Stebila <[stebilad@mcmaster.ca](mailto:stebilad@mcmaster.ca)>.
 
-
-
 License
 -------
 
 liboqs is licensed under the MIT License; see [LICENSE.txt](https://github.com/open-quantum-safe/liboqs/blob/master/LICENSE.txt) for details.  liboqs includes some third party libraries or modules that are licensed differently; the corresponding subfolder contains the license that applies in that case.  In particular:
 
 - `src/crypto/aes/aes_c.c`: public domain
 - `src/crypto/rand_urandom_chacha20/external`: public domain
+- `src/crypto/sha3`: public domain
```

Comment:
I haven't seen a statement on https://github.com/Steppenwolfe65/Misc-C that the code is in the public domain. Do we have authorization to use this code?

Reply:
Acknowledging @Steppenwolfe65 comment below; approving review.